docs/topics/multiplatform/multiplatform-compatibility-guide.md

```diff
@@ -28,308 +33,206 @@ with available Gradle, Xcode, and Android Gradle plugin versions:
 > 
 {style="warning"}
 
-## Deprecated compatibility with Kotlin Multiplatform Gradle plugin and Gradle Java plugins
-
-**What's changed?**
-
-Due to compatibility issues between the Kotlin Multiplatform Gradle plugin and the Gradle plugins
-[Java](https://docs.gradle.org/current/userguide/java_plugin.html),
-[Java Library](https://docs.gradle.org/current/userguide/java_library_plugin.html),
-and [Application](https://docs.gradle.org/current/userguide/application_plugin.html),
-there is now a deprecation warning when you apply these plugins to the same project. The warning also appears when another
-Gradle plugin in your multiplatform project applies a Gradle Java plugin. For example, the [Spring Boot Gradle Plugin](https://docs.spring.io/spring-boot/gradle-plugin/index.html)
-automatically applies the Application plugin.
-In future Kotlin releases, the warning will be increased to an error.
+## Kotlin 2.0.0 and later
 
-We've added this deprecation warning due to fundamental compatibility issues between Kotlin Multiplatform's project model
-and Gradle's Java ecosystem plugins. Gradle's Java ecosystem plugins currently don't take into account that other plugins may:
+This section covers incompatible changes that end their deprecation cycle and come into effect in Kotlin 2.0.0−%kotlinVersion%.
 
-* Also publish or compile for the JVM target in a different way than the Java ecosystem plugins.
-* Have two different JVM targets in the same project, such as JVM and Android.
-* Have a complex multiplatform project structure with potentially multiple non-JVM targets.
+<anchor name="android-target-rename"/>
+### Rename of android target to androidTarget
 
-Unfortunately, Gradle doesn't currently provide any API to address these issues.
+**What's changed?**
 
-We previously used some workarounds in Kotlin Multiplatform to help with the integration of Java ecosystem plugins.
-However, these workarounds never truly solved the compatibility issues, and since the release of Gradle 8.8, these workarounds
-are no longer possible. For more information, see our [YouTrack issue](https://youtrack.jetbrains.com/issue/KT-66542/Gradle-JVM-target-with-withJava-produces-a-deprecation-warning).
+We continue our efforts to stabilize Kotlin Multiplatform. An essential step in this way is to provide first-class
```

Comment:
```suggestion
We continue our efforts to stabilize Kotlin Multiplatform. An essential step in this direction is to provide first-class
```

To me "efforts to stabilize" means that KMP is not stable yet, which may be confusing, because it is. Maybe it's more precise to say "to make it more stable"?